lib/plugins/audit.js

```diff
@@ -6,6 +6,7 @@ var assert = require('assert-plus');
 var bunyan = require('bunyan');
 var HttpError = require('restify-errors').HttpError;
 var VError = require('verror');
+var lodash = require('lodash');
```

Comment:
could you change this line to `var _ = require('lodash');`?
We call lodash `_` in other files in restidf.